packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -24,26 +24,27 @@ import {
 } from "@fluidframework/telemetry-utils";
 import { MockDeltaManager, MockQuorumClients } from "@fluidframework/test-runtime-utils";
 import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
-import { IRequest, IResponse, FluidObject } from "@fluidframework/core-interfaces";
+import { IRequest, IResponse, FluidObject, IProvideFluidRouter } from "@fluidframework/core-interfaces";
 import { ContainerMessageType, ContainerRuntime, IContainerRuntimeOptions } from "../containerRuntime";
 import { PendingStateManager } from "../pendingStateManager";
 import { DataStores } from "../dataStores";
 
 describe("Runtime", () => {
+    const getMockContext = ((): Partial<IContainerContext> => {
```

Comment:
Moved further up in the `describe()` hierarchy so new tests can use it. Had to rename a few in other suites.